receiver/otlpreceiver/config.go

```diff
@@ -66,6 +81,20 @@ func (cfg *Config) Unmarshal(conf *confmap.Conf) error {
 
 	if !conf.IsSet(protoHTTP) {
 		cfg.HTTP = nil
+	} else {
+		// Verify URL path sanity
+		signalURLPaths := []*string{&cfg.HTTP.TracesURLPath, &cfg.HTTP.MetricsURLPath, &cfg.HTTP.LogsURLPath}
+		for i, urlPath := range signalURLPaths {
+			u, err := url.Parse(*urlPath)
+			if err != nil {
+				return errors.New("Invalid HTTP URL path set for signal")
```

Comment:
I think you should wrap the original error, so that users would know what's wrong.

```suggestion
				return fmt.Errorf("invalid HTTP URL path set for signal: %w", err)
```